tools/device_management.py

```diff
@@ -131,10 +134,14 @@ def inner(options):
                 "mbed-cli-auto {}".format(api_key.name),
                 description="cetificate auto-generated by Mbed CLI"
             )
-        LOG.info("Writing developer certificate %s into c file "
+        if getattr(options, 'disable_cert'):
+            LOG.warning("Developer certificate %s not written to "
+                "mbed_cloud_dev_credentials.c", dev_cert_info.name)
```

Comment:
Why would doing exactly what it was told to be a warning? Why not info, like when writing the cert?

Reply:
Yes I thought about it, like should be info, or warning. The thing is, the credentials would need to be written, if we are not writing it then they have to. I thought may be it might get missed under "info" and will be better suited under "warning"

Reply:
That's not how I decide if something is suited for a warning. My first check is "did the user ask for this behavior" if they did, the program is performing as specified and doing that should not  issue a warning, which would indicate that either it could not perform part of it's task or that it encountered minor issues with it's task. If we don't follow this, then we risk teaching users to ignore warnings, which can be an issue when the warnings are telling them something that they would like to know.

Reply:
Noted